src/main/java/dude/DialogBox.java

```diff
@@ -0,0 +1,69 @@
+package dude;
```

Comment:
Maybe the codebase would be better structured if you extracted all the javaFX components to a separate package.